DevCycle/ObjC/ObjCDVCClient.swift

```diff
@@ -76,9 +137,21 @@ public class ObjCDVCClient: NSObject {
     @objc public func variable(key: String, defaultValue: Any) throws -> ObjCDVCVariable {
         var variable: ObjCDVCVariable
         if let variableFromConfig = self.client?.config?.userConfig?.variables[key] {
-            variable = try ObjCDVCVariable(key: key, type: variableFromConfig.type, evalReason: variableFromConfig.evalReason, value: variableFromConfig.value, defaultValue: defaultValue)
+            variable = try ObjCDVCVariable(
```

Comment:
the swift version no longer throws when creating this, it just logs the error that types don't match and sets the value to default value